services/helper/notifications.js

```diff
@@ -0,0 +1,126 @@
+const moment = require("moment-timezone");
```

Comment:
@rakeshSgr  Do we actually required notifications helper in mentoring service ?

Reply:
yes, for now we need it.